### PR TITLE
Economy input vars

### PR DESCRIPTION
**Pull request by paopis**
Included new variables in the economy.yaml file to be able to upload the input data that is used in the macroeconomic models to characterize the different scenarios.

### CODE REVIEW

File: definitions/variable/economy/economy.yaml [hunk 1]
```diff
@@ -212,4 +212,38 @@
     unit: [EUR_2020/t CO2, USD_2010/t CO2]
     skip-region-aggregation: true
 
-
+- Extraction Limit:
```
Comment (danielhuppmann) [inline definitions/variable/economy/economy.yaml:215]: Does it make sense to have one limit across all fossil resources? Shouldn't this be distinguished between different fuels?

Comment (paopis) [reply definitions/variable/economy/economy.yaml:215]: In general, it is true. Nevertheless, we have used a single parameter to control the extraction of oil and gas. I can make three parameters: one for oil and one for gas and use the same parameter in the data upload, though. I will make a new pull request with the updated variables

Comment (danielhuppmann) [reply definitions/variable/economy/economy.yaml:215]: No, do NOT make a new pull request - this is not how GitHub works.

Comment (paopis) [reply definitions/variable/economy/economy.yaml]: Ok. I only updated the description of the variables, then.

File: definitions/variable/economy/economy.yaml [hunk 2]
```diff
@@ -212,4 +212,43 @@
     unit: [EUR_2020/t CO2, USD_2010/t CO2]
     skip-region-aggregation: true
 
+- Extraction Limit for Natural Gas:
```
Comment (danielhuppmann) [inline definitions/variable/economy/economy.yaml:215]:
```suggestion
- Extraction Limit|Natural Gas:
```

File: definitions/variable/economy/economy.yaml [hunk 3]
```diff
@@ -212,4 +212,43 @@
...
+    description: Level of allowed extractions for natural gas, relative to year 2011
+    note: The index equals 1 in 2011, the reference year
+    unit: index
 
+- Extraction Limit for Crude Oil:
```
Comment (danielhuppmann) [inline definitions/variable/economy/economy.yaml:220]:
```suggestion
- Extraction Limit|Crude Oil:
```

**Review by danielhuppmann — COMMENTED** [review]
Two suggestions to use the hierarchy-notation.

**Review by danielhuppmann — APPROVED** [review]
Thanks @paopis!

**Pull request merged**